CHANGELOG.md

```diff
@@ -18,6 +19,7 @@
 
 #### Minor
 
+* Render a text filter instead of a select for large associations (opt-in) [#5548] by [@DanielHeath]
```

Comment:
You left this line here unintentionally, it seems.

Reply:
Oops - fixed.